package.json

```diff
@@ -60,16 +60,17 @@
   "scripts": {
     "build:dedupe-types": "node tools/dedupe-types.js dist/cjs/index.cjs dist/esm/index.js",
     "build:cts": "node -e \"fs.copyFileSync('dist/esm/index.d.ts', 'dist/cjs/index.d.cts')\" && node tools/update-cts.js dist/cjs/types.cts dist/cjs/index.d.cts",
-    "build": "rollup -c && npm run build:dedupe-types && tsc -p tsconfig.esm.json && npm run build:cts",
+    "build:rules": "node tools/build-rules.js",
+    "build": "npm run build:rules && rollup -c && npm run build:dedupe-types && tsc -p tsconfig.esm.json && npm run build:cts",
     "build:readme": "node tools/update-readme.js",
     "build:update-rules-docs": "node tools/update-rules-docs.js",
     "build:baseline": "node tools/generate-baseline.js",
-    "test:jsr": "npx jsr@latest publish --dry-run",
-    "pretest": "npm run build",
+    "prepare": "npm run build",
```

Comment:
This was `pretest` previously. Intentionally changed?

Reply:
The reason we had it `pretest` is because it's easy to forget to rebuild before running `npm test`, and in that case, you may end up chasing down errors that don't exist because you forgot.